site/source/tutorials/006_containers_and_injecting_state/index.md

```diff
@@ -41,4 +43,4 @@ To do this we need to import the `BaseInjector` and the `Injector` mixin that ca
 
 ## Summary
 
-Summarise lift state outside of the widget components, registering an injector with an context that can hold state and/or a functional API and swapping out the existing widgets for the newly created containers.
\ No newline at end of file
+Summarise lift state outside of the widget components, registering an injector with an context that can hold state and/or a functional API and swapping out the existing widgets for the newly created containers.
```

Comment:
This is a WIP post that's not displayed yet. I left the sentence as is because I'm not sure what the intention is but I fixed the spelling of "summarize". I'm not sure what changed about this in the first place, that's my bad as it shouldn't have been in the change set.